src/octopeer-github/test/content/ElementEventBinding/ScrollElementEventBindingTest.ts

```diff
@@ -0,0 +1,59 @@
+import Spy = jasmine.Spy;
+/**
+ * Created by Mitchell on 25-5-2016.
+ * Behaviour unit tests for the ScrollIntoViewElementEventBinding class.
+ */
+
+const esbFactory = new ElementSelectionBehaviourFactory();
+const eebFactory = new ElementEventBindingFactory();
+
+let binder: ElementEventBinding;
+let database: DatabaseAdaptable = new ConsoleLogDatabaseAdapter();
+let selector: ElementSelectionBehaviour;
```

Comment:
So you would prefer multiple statements that are exactly the same? Not very DFY.